resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/DescriptionField.tsx

```diff
@@ -0,0 +1,96 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import type { FC } from 'react';
+import { useFormContext } from 'react-hook-form';
+import TextareaAutosize from 'react-textarea-autosize';
+
+import {
+  BaseFormControl,
+  BaseFormDescription,
+  BaseFormField,
+  BaseFormItem,
+  BaseFormLabel,
+  BaseFormMessage,
+} from '@/common/components/+vendor/BaseForm';
+import { baseTextareaClassNames } from '@/common/components/+vendor/BaseTextarea';
+import { Trans } from '@/common/components/Trans';
+import { cn } from '@/utils/cn';
+
+import type { CreateAchievementTicketFormValues } from './useCreateAchievementTicketForm';
+
+export const DescriptionField: FC = () => {
+  const { t } = useLaravelReactI18n();
+
+  const form = useFormContext<CreateAchievementTicketFormValues>();
+
+  const [description] = form.watch(['description']);
+
+  const showTriggerWarning =
+    description.length < 25 && /(n'?t|not?).*(work|trigger)/gi.test(description);
```

Comment:
There seems to be something else checking for a minimum length without providing any feedback. If I just type in something like "Please fix this", there's no feedback at all, and the Submit button does nothing.

Reply:
Looks like validation is kicking in correctly but not surfacing a visual label. I've adjusted this in latest. On submit, the validation error should now correctly be displayed:

![Screenshot 2024-11-06 at 5 30 13 PM](https://github.com/user-attachments/assets/dc28f9d2-06ae-4d8f-8bf2-16328d7a42e1)

Reply:
"this achievement doesn't work." doesn't show any error message and doesn't submit.

Reply:
Looks like I can't submit long strings of keyboard vomit either.

Reply:
I believe both issues are now addressed in latest:

* "this achievement doesn't work" should now display the proper validation error.
* Required fields were not displaying validation errors, which I why I suspect the submit on keyboard vomit appeared to do nothing. These fields now display validation errors if values are missing.